powerauth-webflow-authentication-login-sca/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/method/loginsca/controller/LoginScaController.java

```diff
@@ -260,4 +303,36 @@ public AuthStepResponse cancelAuthentication() throws AuthStepException {
         }
     }
 
+    /**
+     * User was successfully authenticated using client TLS certificate. Move user to the next step.
+     * @return Login SCA authentication response.
+     */
+    private LoginScaAuthResponse authorizeUsingTlsCertificate(String operationId, String userId, String organizationId) {
+        List<AuthInstrument> authInstruments = Collections.singletonList(AuthInstrument.CLIENT_CERTIFICATE);
+        try {
+            UpdateOperationResponse updateResponse = authorize(operationId, userId, organizationId, authInstruments, null);
+            final LoginScaAuthResponse response = new LoginScaAuthResponse();
+            response.setResult(AuthStepResult.CONFIRMED);
+            response.setMessage("authentication.success");
+            response.getNext().addAll(updateResponse.getSteps());
+            for (AuthStep step: updateResponse.getSteps()) {
+                System.out.println("Next step: " + step.getAuthMethod());
```

Comment:
Forgotten `System.out`, we should have some static check on our files.

On production the operation team is usually really angry to see this in their system logs.

Reply:
Thanks for catching this one, it was used during debugging.